spopt/locate/p_median.py

```diff
@@ -806,8 +806,10 @@ def _from_sparse_matrix(self) -> None:
 
         # Client assignment integer decision variables
         row_indices, col_indices, values = find(self.aij)
+
+        # TODO: add `strict=True` back in to `zip` – gh-spopt#402
```

Comment:
I'd remove this. We're zipping an output of scipy.sparse.find. That is never of different length so using strict here has no reason.

Reply:
Good point. I'll remove the note and add a `# noqa`

Reply:
Why `noqa`?

Reply:
> Why `noqa`?

[`B905`](https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/) is raised without it.

Reply:
@martinfleis Is there something I am missing that makes this a bad idea?

Reply:
> [B905](https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/) is raised without it.

Is it? I don't see it locally and it should not, given we specify `target-version = "py38"`. Ruff should not suggest to include a keyword that does not exist in 3.8.

Reply:
It doesn't matter in practice, it is just strange that you see this. I don't mind to have it there, I just think that right now it should not raise and once we update min Python, Ruff will tell us to add a keyword here. But not now.

Reply:
Fair enough. I am being proactive with it. No need to do so, I suppose. I'll remove and merge.